README.md

```diff
@@ -226,7 +226,15 @@ the `prepare_virtual_env` scripts.
 
 Suggested IDE: [PyCharm Community](https://www.jetbrains.com/pycharm/download/)
 
-_*If using Visual Studio Code, make sure to set `"qtForPython.uic.liveExecution.enabled": false` when using the QT extension._
+## Visual Studio Code
+
+Clone this repository and open the folder in Visual Studio Code. It suggest several useful plugins for developing which you should download and install.
+
+If your python is setup properly, you can use the `Create venv with all exporters` task by pressing CTRL+SHIFT+P, type in `Task` to find + select `Tasks: Run task` and then select the task. It will create the venv with all requirements for you.
```

Comment:
It is "find and select" but I don't like "find and select [...] and" because it reads kinda weird. 
"type in `Task` to find, select [...]" reads like I missed words because the first part is not a full sentence.
I will just remove "to find"